assets/tests/components/map.test.tsx

```diff
@@ -70,6 +72,14 @@ const vehicle: Vehicle = vehicleFactory.build({
   crowding: null,
 })
 
+jest.mock("appData")
+
+beforeEach(() => {
+  ;(appData as jest.Mock).mockImplementation(() => ({
+    userTestGroups: JSON.stringify([]),
+  }))
+})
```

Comment:
nit(non-blocking): what if instead of using appData we mocked the `inTestGroup` function to return `true`?

If that's not possible, or if we want to be testing the actual test group string we're using, then I think we should mock `getTestGroups` to return an array, saving us a JSON "serde"(serialize-deserialize)?

I'm conflicted though because that feels like depending on the test groups utility API too specifically.

Reply:
I'd probably lean slightly in the direction of mocking the `getTestGroups` function as well.

Reply:
Thanks for the suggestion! Addressed in [cadf169](https://github.com/mbta/skate/pull/1824/commits/cadf169e9b18137c2f2b3c287bbd008ae008d48a)

Reply:
Ah I tried mocking `getTestGroups` but it ran into issues since it is called by `inTestGroups` -- found this [post](https://medium.com/@DavideRama/mock-spy-exported-functions-within-a-single-module-in-jest-cdf2b61af642) about possible solutions (the better one to me seems like putting `getTestGroups` into its own file, but I'm not sure if that is overkill and only opted to mock `inTestGroup` for now

Reply:
Thank you for finding that out for me AND linking relavent info @KaylaBrady!

I'm agreeing with Eddie that mocking `getTestGroups` is better, especially if we ever needed to test code where one test group is on and the other is off. If to achieve that we have to move it into it's own file?... I don't mind, but I agree it feels a little overkill. 
When originally writing this comment I was hoping jest would have some easy way to make it work :c

Reply:
Yeah I'm with you both that `getTestGroups` would be better here, and might as well do it now so it is easier for testing test groups in the future! I'll add a commit for it.

Reply:
updated in [9afa4d0](https://github.com/mbta/skate/pull/1824/commits/9afa4d08600ce7dd57a4ab833eff022bddc0e12f)!

Reply:
I love it!! this is great thank you!